### PR TITLE
App does not work in flutter apps

### LINKED ISSUE #127: Speak Touch does not work in flutter apps
## Describe the bug:

Apparently our beloved screen reader does not work with apps made in Flutter. I did an initial investigation and it seems to be related to the `isImportantForAccessibility` flag

## To Reproduce:

Find an app made in Flutter or make a test app in Flutter. I tested with [Memo](https://play.google.com/store/apps/details?id=com.olmps.memoClient).

## More infos:
App version: 1.0.0 - DEV
Android version: Irrelevant
Device: Irrelevant

### PR DESCRIPTION
**Pull request by Irineu333**
### Description

Read the issue.

### Fixed issues
- [App does not work in flutter apps](https://github.com/NeoA11y/SpeakTouch/issues/127)

### CODE REVIEW

**Review by PatrykMis — APPROVED** [review]
Well done. Tested extensively and no issues. It works like in other non-Flutter apps.

**Comment by Irineu333** [comment]
I only tested it on [Memo](https://play.google.com/store/apps/details?id=com.olmps.memoClient), I will test it on more Flutter apps:
- [x] [99](https://play.google.com/store/apps/details?id=com.taxis99) - Found bugs
- [x] [Play Console](https://play.google.com/store/apps/details?id=com.google.android.apps.playconsole) - Found bugs
- [x] [Precifica.me](https://play.google.com/store/apps/details?id=com.infinitystudios.precificame) - Found bugs
- [x] [Rester](https://play.google.com/store/apps/details?id=br.tiagohm.restler) - Found bugs
- [x] [Serasa](https://play.google.com/store/apps/details?id=br.com.serasaexperian.consumidor)
- [x] [Nubank](https://play.google.com/store/apps/details?id=com.nu.production)
- [x] [Simpler](https://play.google.com/store/apps/details?id=ru.zengalt.simpler) - Found bugs
- [x] [Webmotors](https://play.google.com/store/apps/details?id=hands.android.webmotors)  - Found bugs

**Comment by Irineu333** [comment]
@coderabbitai review

File: app/src/main/java/com/neo/speaktouch/model/NodeFilter.kt [hunk 1]
```diff
@@ -30,11 +30,11 @@ sealed interface NodeFilter {
         override fun filter(node: AccessibilityNodeInfo): Boolean {
             val compat = AccessibilityNodeInfoCompat.wrap(node)
 
-            if (!NodeValidator.isValidForAccessibility(compat)) return false
+            if (!compat.isVisibleToUser) return false
```
[the remark this answers is not in the document]
Comment (Irineu333) [reply app/src/main/java/com/neo/speaktouch/model/NodeFilter.kt:33]: It only makes an obvious suggestion, I'm disappointed.

**Comment by Irineu333** [comment]
@coderabbitai resolve 
I expected more from this tool. The comments are basically: "you changed something, check if it's correct".

**Comment by CodeRabbitSupport** [comment]
> @coderabbitai resolve I expected more from this tool. The comments are basically: "you changed something, check if it's correct".

@Irineu333 There was a slight degradation in our noise suppression, but it has since been fixed.

**Pull request merged**